.buildkite/pipeline.yml

```diff
@@ -84,6 +84,15 @@ steps:
         gopath-checkout#v1.0.1:
           import: github.com/m3db/m3
     <<: *common
+  - label: "FOSSA license scan"
+    command: make clean install-vendor fossa
+    plugins:
```

Comment:
Can we try the new soft_fail feature here too so these failures don't block docs / docker builds?
```yaml
soft_fail:
- exit_status: "*"
```

Reply:
Hm, as we discussed - I'd rather not until it starts to flake.  Before we experienced issues with the request size upload FOSSA was very reliable.

Looks like the issues have been resolved.